sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -416,18 +417,23 @@ public Node visitInsert(SqlBaseParser.InsertContext context) {
 
         final Insert.DuplicateKeyType duplicateKeyType;
         final List<SqlBaseParser.AssignmentContext> assignment;
+        final List<String> conflictColumns;
```

Comment:
Would it make sense to have a `List<QualifiedName>` here? We use that usually in other places in the parser/AST when we've a `qnames`

Reply:
Hmm, this gets kind of ugly in the code. The qname is kind of an exception due to the object notation. Otherwise, we don't support qnames for columns names in INSERT statements. I would keep it as a string because otherwise I have to change the code to deal with both `List<String>` and `List<QualifiedName>`, the latter being `List<List<String>>`.